src/geography.cpp

```diff
@@ -410,7 +496,12 @@ void init_geography(py::module &m) {
         Parameters
         ----------
         geography : :py:class:`Geography` or array_like
-            Geography object(s)
+            Geography object(s).
+
+        Returns
+        -------
+        prepared : Geography or array
+            The same input Geography object(s) with an attached index.
```

Comment:
When adding this, it might be worth explicitly mentioning in the docstring higher up that the function also updates the geographies in place, and not only the ones that are returned